requirements-dev.txt

```diff
@@ -5,8 +5,8 @@ flake8==3.7.9
 nose==1.3.7
 splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-datarequests@training#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@develop#egg=ckanext-data_qld
```

Comment:
todo, revert back to qld-gov-au once merged.

Reply:
@duttonw - I have updated this via commit https://github.com/MarkCalvert/ckanext-ytp-comments/commit/036112d3f95d8b4f99b9471c3c6c0d42903dd04c